pyproject.toml

```diff
@@ -20,7 +20,8 @@ dependencies = [
     'pydantic>=2.0',
     'pydantic-settings>=2.0',
     'aind-data-schema-models==0.1.1',
-    'aind-slurm-rest==0.1.0'
+    'aind-slurm-rest==0.1.0',
+    'pyyaml'
```

Comment:
If pyyaml is only being used in the testing directory, it might be better to put in the optional dev dependencies.

Reply:
You can also add a method to instanitate the class from a yaml file path if that's going to be a common workflow. There are a couple of ways to go about doing that.

Reply:
Moved pyyaml to dev build